install_seafile.sh

```diff
@@ -3,35 +3,30 @@ set -e
 set -x
 set -o pipefail
 
-SEAFILE_DIR="/opt/seafile/seafile-server-${SEAFILE_VERSION}"
+SEAFILE_DIR='/var/lib/seafile/scripts'
 SEAHUB_DIR="$SEAFILE_DIR/seahub"
 
-# backup original seahub's thirdpart folder, because the github clone doesn't
-# have it (and we haven't investigate in how to build it)
-mv "$SEAHUB_DIR/thirdpart" /tmp
-
-# remove original seahub or we won't be able to clone ours there
-rm -rf "$SEAHUB_DIR"
-
-# Use open365's seahub from github
-git clone --depth 1 --branch open365 https://github.com/Open365/seahub.git "$SEAHUB_DIR"
-
-# put thirdpart folder inside our seahub
-rm -rf "$SEAHUB_DIR/thirdpart"
-mv /tmp/thirdpart "$SEAHUB_DIR/thirdpart"
-
-# compile and minify seahub (translations and js). We cannot do 'make dist'
-# when building the image because there are some steps that must be done after
-# setting up seafile, because it depends on ccnet's config and some envars.
-(
-	cd "$SEAHUB_DIR"
-	apt-get install -y gettext
-	npm install --global requirejs
-	export PYTHONPATH=$PWD/../seafile/lib64/python2.6/site-packages:$PWD/thirdpart:$PWD
-	export PATH=$PATH:thirdpart/Django-1.5.12-py2.6.egg/django/bin
-	make locale uglify
-	apt-get autoremove -y gettext
-)
+cd /root/
+BASE_URL="https://s3-eu-west-1.amazonaws.com/apk-packages"
+
+wget \
+	${BASE_URL}/ccnet-5.0.5-r0.apk \
+	${BASE_URL}/ccnet-dev-5.0.5-r0.apk \
+	${BASE_URL}/ccnet-libs-5.0.5-r0.apk \
+	${BASE_URL}/libsearpc-3.0.7-r0.apk \
+	${BASE_URL}/libsearpc-dev-3.0.7-r0.apk \
+	${BASE_URL}/libevhtp-1.2.9-r0.apk \
+	${BASE_URL}/py-ccnet-5.0.5-r0.apk \
+	${BASE_URL}/py-libsearpc-3.0.7-r0.apk \
+	${BASE_URL}/py-seafile-5.0.5-r1.apk \
+	${BASE_URL}/seafile-5.0.5-r1.apk \
+	${BASE_URL}/seafile-common-5.0.5-r1.apk \
+	${BASE_URL}/seafile-dev-5.0.5-r1.apk \
+	${BASE_URL}/seafile-server-5.0.5-r1.apk \
+	${BASE_URL}/seafobj-5.0.5-r1.apk
+
+apk add --no-cache --allow-untrusted /root/*.apk
```

Comment:
instead of downloading all these packages, couldn't you create an `apk` repo? This way you won't have problems in the future with version numbers (I see there are some packages that are `-r0` and some others that are `-r1`).